src/main/generated/resources/data/minestuck/advancements/recipes/minestuck/chessboard_white.json

```diff
@@ -1,43 +1,43 @@
 {
-  "parent": "minecraft:recipes/root",
```

Comment:
Hmm, I didn't notice these last time around. Same problem as back with the lang file. Deleting all generated resources and then rerunning runData should fix it

Reply:
done